pvsite_datamodel/read/model.py

```diff
@@ -0,0 +1,52 @@
+""" Read functions for getting ML models. """
+import logging
+from typing import Optional
+
+from sqlalchemy.orm import Session
+
+from pvsite_datamodel.sqlmodels import MLModelSQL
+
+logger = logging.getLogger(__name__)
+
+
+def get_model(session: Session, name: str, version: Optional[str] = None) -> MLModelSQL:
```

Comment:
I wonder if this should be called "get_or_create_model" just to be really explicit that it isn't a read only operation as a standalone "get" would suggest?

Reply:
thats a good idea